README.md

```diff
@@ -44,38 +50,52 @@ command line like this:
 ./normalizer < sample.csv > output.csv
 ```
 
+### JSON (If you want us to run your code from a browser):
+
+Please write a static webpage that normalizes and displays the data contained in
```

Comment:
"static webpage"  confuses me a bit.  This could also be a SPA right?

Reply:
Yes but even a SPA is static if it's not being dynamically rendered by a server. All I meant by this that they can give us an index.html file with static assets and not worry about running any kind of a web server.